notebooks/reactivex.io/startup.py

```diff
@@ -88,7 +88,7 @@ def on_completed(self)   : return self._on('cmpl', 'fin')
 def subs(src, **kw):
     # required for e.g. .multicast:
     obs = Subscriber(src, **kw)
-    subscription = src.subscribe(obs)
+    subscription = obs.subscribe_to(src)
```

Comment:
Revert?

Reply:
Ah, yes, that's not right. Thanks!

And actually Subscriber isn't strictly speaking a subclass of `Observer` here, so I changed it to just pass the callbacks from `obs` to `src.subscribe`.

My impressions is the notebooks need to still be aligned to v3, anyway, is that correct?

I've amended, and also found another quirk in delaywithmapper (I think it was broken before, apparently not covered by tests).